packages/app/test/firebaseApp.test.ts

```diff
@@ -87,6 +87,32 @@ function executeFirebaseTests(): void {
       assert.equal(registrations, 2);
     });
 
+    it('Will do nothing if registerService is called again with the same name', () => {
+      let registrations = 0;
+      (firebase as _FirebaseNamespace).INTERNAL.registerService(
+        'test',
+        (app: FirebaseApp) => {
+          registrations += 1;
+          return new TestService(app);
+        }
+      );
+      firebase.initializeApp({});
+      assert.equal(registrations, 0);
+      (firebase as any).test();
+      assert.equal(registrations, 1);
+
+      (firebase as _FirebaseNamespace).INTERNAL.registerService(
+        'test',
+        (app: FirebaseApp) => {
+          registrations += 1;
+          return new TestService(app);
+        }
+      );
+
+      (firebase as any).test();
```

Comment:
We should just test no exception is thrown the second time and that `firebase.test === the previous firebase.test`

Reply:
Added equality test.  To check for no exception, do we have to do anything?  Or just leave it alone and throwing will fail the test?

Reply:
Updated test to check for exception and service namespace equality.